src/braket/experimental/autoqasm/doc/decorators.md

```diff
@@ -50,11 +50,9 @@ def bell(q0: int, q1: int) -> None:
 def two_bell() -> None:
     bell(0, 1)
     bell(2, 3)
-
-two_bell_program = two_bell()
 ```
 
-Let's take a look at the serialized output from `two_bell_program.to_ir()`, which shows that the modularity of the subroutine is preserved.
+Let's take a look at the serialized output from `two_bell.to_ir()`, which shows that the modularity of the subroutine is preserved.
```

Comment:
Optional refactor to use `display`, though it's probably worth having `to_ir` somewhere in our docs too

Reply:
Agreed, I converted the other two to `display()`, I'll leave this one as `to_ir()` for now.